tests/python/PycheckerBearTest.py

```diff
@@ -0,0 +1,51 @@
+from queue import Queue
+import os.path
+
+from coalib.settings.Section import Section
+from coalib.results.Result import Result
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.BearTestHelper import generate_skip_decorator
+
+from bears.python.PyCheckerBear import PyCheckerBear
```

Comment:
Here you have imported `PyCheckerBear` from `bears.python.PyCheckerBear` but that file doesn't exist

Reply:
yup thats writing error i will correct it